grails-app/controllers/org/bbop/apollo/GroupController.groovy

```diff
@@ -177,7 +180,7 @@ class GroupController {
         UserGroup group = new UserGroup(
                 name: dataObject.name
         ).save(flush: true)
-        def currentUser = permissionService.currentUser
+        def currentUser = User.findByUsername(dataObject.username)
```

Comment:
I would leave it as-is unless there has been a problem with the code.    The user should be bound to the code uness you've got bigger problems, so it would be better to catch that.

Reply:
`def currentUser = permissionService.currentUser` works if there's a login user from the UI, but as for API call, it failed. Let me know if I am doing something wrong.